src/html/shared/signup.html

```diff
@@ -4,12 +4,15 @@
             <div class="o-layout__item  u-push-1/12@tablet  u-4/12@tablet">
                 <span class="c-logo  c-logo--huge">{{sprite("logo","0 0 3941 1024")}}</span>
                 <h3 class="c-signup__heading">Sign me up for more information</h3>
-                <form action="submit" class="o-form">
+                <form action="https://github.us17.list-manage.com/subscribe/post" method="POST" class="o-form" data-module="Form">
+                    <input type="hidden" name="u" value="43373efd1b02bff2a76a8ce96">
+                    <input type="hidden" name="id" value="58ed49b77c">
+
                     <label class="u-hidden-visually">Sign up with email address</label>
-                    <input type="email" placeholder="Your email address" class="o-form__input" required>
-                    <button class="c-btn  c-btn--secondary  c-btn--small o-form__submit">Sign up</button>
+                    <input id="email" name="MERGE0" type="email" placeholder="Your email address" class="o-form__input" data-module="Form" required>
```

Comment:
don't think data-module="Form" should be used here. Best remove `data-module` and select the element within the `Form` module.

so something like

`this.el.querySelector("#email").addEventListener()`